src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
-public class MainApp {
+import core.basesyntax.garage.Bulldozer;
+import core.basesyntax.garage.Excavator;
+import core.basesyntax.garage.Machine;
+import core.basesyntax.garage.Truck;
 
+public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = new Machine[]{truck, bulldozer, excavator};
+        for (Machine machine : machines) {
+            machine.doWork();
```

Comment:
The message inside the 'doWork' method should be informative and indicate what type of Machine is working. Make sure that the implementation of this method in the subclasses of Machine provides a clear message that includes the type of machine, such as 'Truck', 'Bulldozer', or 'Excavator'.